.github/actions/build/action.yml

```diff
@@ -0,0 +1,21 @@
+name: "Build"
+inputs:
+  package-name:
+    description: 'Name of the package'
+    required: true
+runs:
+  using: "composite"
+  steps:
+    - name: Setup Node.js
+      uses: actions/setup-node@v3
+      with:
+        node-version-file: .nvmrc
+
+    - name: Setup environment
+      uses: ./.github/actions/setup
```

Comment:
maybe I'm missing something, but where is the "setup" action inside `.github/actions`?

Reply:
It's already in the main branch